gdx/src/com/badlogic/gdx/maps/tiled/renderers/IsometricStaggeredTiledMapRenderer.java

```diff
@@ -28,20 +28,36 @@
 
 public class IsometricStaggeredTiledMapRenderer extends BatchTiledMapRenderer {
 
+	private float unitScaleY;
```

Comment:
Since it's up for review, I decided to modify the code to fit my use case needs i.e. Isometric. Other tile maps predominantly use equivalent values for tile length and width, especially if a user intends to match a map up to a grid or graph, while keeping things as simple as possible.

Isometric maps are different because tiles with differing values for length and width, and tile sets that use only one value for these, are represented roughly equally. You need to supply the proper unit scales for the map to be centered on the default view, and match up to world coordinates.

As for making it private, all the other fields in the class are marked so, and the intent for each is to be used internally. 

Furthermore, in this case, since the right unit scale is coupled to a tile's pixel dimensions, there is no use to expose it for modification, unless the tiles' dimensions itself, is modified. Setting the value for unit scale is usually down outside the creation of the renderer, where one can infer the tile dimensions, and compute the value for unit scale.